TestCases/optimization_euler/equivalentarea_naca64206/NACA64206_adjoint.cfg

```diff
@@ -0,0 +1,153 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: NACA64-206 in inviscid, supersonic flow                  %
+% Author: Yuki Utsumi	      	      	              	       	       %
+% Institution: Individual				               %
+% Date: 2021.07.19                                                             %
+% File Version 7.1.1 "Blackbird"                                                %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Physical governing equations (EULER, NAVIER_STOKES)
+SOLVER= EULER
+%
+% Mathematical problem (DIRECT, CONTINUOUS_ADJOINT, DISCRETE_ADJOINT)
+MATH_PROBLEM= DISCRETE_ADJOINT
```

Comment:
just as MATH_PROBLEM=DIRECT this line is not necessary and the Discrete adjoint mode is automatically used when you call the file using SU2_CFD_AD
Edit: Just as pedro mentioned, if this option is dividing the two cfg's then you can throw this option out and use the same cfg for both: direct and adjoint